packages/web-app-files/src/helpers/contextualHelpers.js

```diff
@@ -17,6 +17,13 @@ export const shareInviteCollaboratorHelp = {
     $gettext('Invited persons can not see who else has access')
   ]
 }
+export const shareInviteCollaboratorHelpCern = {
+  list: [
```

Comment:
do you want to change it or merge?

Reply:
Initially it was just an array, but I later changed it into a list field in case we have to add more stuff. I also added the logic to merge both lists in [here](https://github.com/owncloud/web/pull/7034/files#diff-b1e39cb16b02dbdbc6df65c62c56e76016411650bd79119fe63da088843cd288R155) for this.

Reply:
Ah, missed that part. cool 😎